trustyai_tests/tests/drift/test_drift.py

```diff
@@ -27,6 +28,8 @@ class TestDriftMetrics:
     """
 
     def test_gaussian_credit_model_metadata(self, model_namespace, trustyai_service, gaussian_credit_model):
+        wait_for_model_pods_registered(namespace=model_namespace)
```

Comment:
Could we rename this at some point to tell that it's waiting specifically for ModelMesh pods?
We will have to support KServe in the test at some point too.

Reply:
Thanks for your comment, will rename the function in a separate PR.